scripts/Dockerfile.base

```diff
@@ -26,9 +26,7 @@ RUN apk add --no-cache \
 # Terraform was disabled in the edge repo due to a build issue.
 # https://gitlab.alpinelinux.org/alpine/aports/-/commit/f3e263d94cfac02d594bef83790c280e045eba35
 # Using wget for now. Note that busybox unzip doesn't support streaming.
-#
-# WARNING: Do not update to 1.6.x, as it is the first release licensed under BSL instead of MPL.
```

Comment:
We're OK with this from a legal standpoint?

Reply:
> We're OK with this from a legal standpoint?

I'm assuming so based on @bpmct 's wording [here](https://github.com/coder/coder/issues/12009#issue-2117436345) but good to super confirm.

Reply:
Yeah we're good